### ISSUE TITLE
Cannot read properties of undefined (reading 'data') adyen-cc-method.js:244:67

### ISSUE THREAD

**Issue by kayintveen** [Adyen/adyen-magento2#1461]
We are trying to upgrade our Magento 2.4.3 installation from version Adyen version 6.7.6 towards 8.2.0 but we also tried 7.3.7. In both cases, everything looks good until we reach the credit card payment. Upon clicking the radio button to select the payment method we get the error `Uncaught TypeError: Cannot read properties of undefined (reading 'data')` on `adyen-cc-method.js:244:67`

This is the following line
```
    let stateData = JSON.stringify(this.cardComponent.data);
```

It's clear to me that this.cardComponent is undefined. 

**To Reproduce**
1. Go to checkout and try to select payment method
2. Click on radio button (it does not respond
3. See error

**Expected behavior**

**Magento version**
2.4.3-p1

**Plugin version**
8.2.0

**Screenshots**
![image](https://user-images.githubusercontent.com/1884283/163246669-120b2423-1908-4393-8c1c-1ead6128a613.png)
![image](https://user-images.githubusercontent.com/1884283/163246685-4326c9e4-3a81-42af-9165-9e6609f9536b.png)

**Comment by jonathanribas** [comment]
Seems to be a duplicate of https://github.com/Adyen/adyen-magento2/pull/1456#issuecomment-1096888675

**Comment by tnaber** [comment]
HI @kayintveen,

A temporary fix is proposed in #1456, we would greatly appreciate it if you would be able to verify whether this works for you.

Cheers,
Titus

**Comment by tnaber** [comment]
Closing this as it is a duplicate, feel free to check out the updated fix in #1456!
